keyboards/anomalykb/a65i/readme.md

```diff
@@ -0,0 +1,15 @@
+# a65i
+
+![a65i](https://i.imgur.com/J4fzVVt.jpg)
+
+PCB designed by Lfgberg for the AnomalyKB a65i
+
+* Keyboard Maintainer: [Lfgberg](https://github.com/lfgberg)
+* Hardware Supported: A65I PCB
+* Hardware Availability: [AnomalyKB](anomalykb.co)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader to flash new firmware?